src/com/company/Human.java

```diff
@@ -0,0 +1,132 @@
+package com.company;
+
+/*Есть два рода людей: женщины и мужчины. Они обладают одинаковым набором свойств за небольшим исключением (у кого как) и методов.
+        Пусть будет определен следующий набор свойств:
+        - признак пола (boolean, true - мужчина, false - женщина)
+        - имя (String)
+        - фамилия (String)
+        - рост (float)
+        - вес (float)*/
+
+//variables
+public abstract class Human {
+    private boolean gender;
+    private String name;
+    private String surName;
+    private float height;
+    private float weight;
+
+    //constructor
+    public Human(boolean gender, String name, String surName, float height, float weight) {
+        this.gender = gender;
+        this.name = name;
+        this.surName = surName;
+        this.height = height;
+        this.weight = weight;
+    }
+
+    @Override
+    public String toString() {
+        return "{" +
+                "gender=" + gender +
+                ", name='" + name + '\'' +
+                ", surName='" + surName + '\'' +
+                ", height=" + height +
+                ", weight=" + weight +
+                '}';
+    }
+
+    //getter
+    public boolean isGender() {
+        return gender;
```

Comment:
i would remove isGender method and create two new methods isWoman and isMan.

Reply:
`public boolean isMan() {
        return gender;
 }
public boolean isWoman() {
        return !gender;
 }
`

Reply:
Thank you)